Turf/Turf.swift

```diff
@@ -285,3 +288,67 @@ public struct Polyline {
         return closestCoordinate
     }
 }
+
+public struct Polygon {
+    var polygonCoordinates: [[CLLocationCoordinate2D]]
+    
+    var polygonArea: Double {
+        var area:Double = 0
+   
+        if (!polygonCoordinates.isEmpty && polygonCoordinates.count > 0) {
+            
+            area += abs(ringArea(polygonCoordinates[0]))
+
+            for coordinate in polygonCoordinates.suffix(from: 1) {
+                area -= abs(ringArea(coordinate))
```

Comment:
Note that each item in `polygonCoordinates` represents a ring’s coordinates, not an individual `CLLocationCoordinate2D`.

A more functional approach would be:

```swift
let area = abs(ringArea(polygonCoordinates.first ?? []))
    - polygonCoordinates.suffix(from: 1)
        .map { abs(ringArea($0)) } // convert the inner rings to their areas
        .reduce(0, +) // sum the inner areas
```

Reply:
If we use separate `outerRing` and `innerRings` properties as in https://github.com/mapbox/turf-swift/pull/21#discussion_r155851566, then this simplifies further to:

```swift
let area = abs(ringArea(outerRing))
    - innerRings
        .map { abs(ringArea($0)) } // convert the inner rings to their areas
        .reduce(0, +) // sum the inner areas
```